src/test/logging_tests.cpp

```diff
@@ -200,7 +200,9 @@ BOOST_FIXTURE_TEST_CASE(logging_Conf, LogSetup)
         const char* argv_test[] = {"bitcoind", "-loglevel=debug"};
         std::string err;
         BOOST_REQUIRE(args.ParseParameters(2, argv_test, err));
-        init::SetLoggingLevel(args);
+
+        auto result = init::SetLoggingLevel(args);
+        BOOST_REQUIRE(result);
```

Comment:
If you're touching unit tests anyway, why not add a test for invalid args here as well?